js/config.ts

```diff
@@ -85,7 +85,7 @@ const DEFAULT_CONFIG = {
   oceanDensity: 1,
   continentDensity: 3,
   // How fast continent is stretching along divergent boundary. Bigger value means it would turn into ocean / see faster.
-  continentalStretchingRatio: 8,
+  continentalStretchingRatio: 4,
```

Comment:
Out of curiosity, should the second-to-last word in the comment in line 87 be "sea"?

Reply:
It should. :) I'll fix it.